plugin-maven/src/main/java/com/diffplug/spotless/maven/sql/Sql.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.maven.sql;
+
+import java.util.Collections;
+import java.util.Set;
+
+import com.diffplug.spotless.maven.FormatterFactory;
+
+/**
+ * A {@link FormatterFactory} implementation that corresponds to {@code <dbeaver>...</dbeaver>} configuration element.
+ * <p>
+ * It defines a formatter for sql source files.
+ */
+public class Sql extends FormatterFactory {
+	private static final String LICENSE_HEADER_DELIMITER = null;
+
+	@Override
+	public Set<String> defaultIncludes() {
+		return Collections.emptySet();
+	}
+
+	@Override
+	public String licenseHeaderDelimiter() {
+		return LICENSE_HEADER_DELIMITER;
```

Comment:
Better to just return `null`, rather than `LICENSE_HEADER_DELIMITER = null`.  That's our fault, not yours, you are following the same pattern [from here](https://github.com/diffplug/spotless/blob/main/plugin-maven/src/main/java/com/diffplug/spotless/maven/typescript/Typescript.java).  Can you fix `Typescript` in this PR too so that other languages without a default delimiter don't make the same mistake?